api-reference/upload-file-api/client-side-file-upload.md

```diff
@@ -344,12 +245,32 @@ import { IKImage, IKContext, IKUpload } from 'imagekitio-react'
 function App() {
   const publicKey = "your_public_api_key";
   let urlEndpoint = "https://ik.imagekit.io/your_imagekit_id";
```

Comment:
IKImage import and urlEndpoint is not needed in this example

Reply:
`urlEndpoint` is required for upload on sdk I have removed IKImage